native_client/java/DeepSpeech-jni/README.MD

```diff
@@ -0,0 +1,104 @@
+# DeepSpeech JNI Library
+
+This library acts as a layer between the DeepSpeech native
+library and the Java bindings. This library contains the definitions
+of the native functions declared as `native` in the DeepSpeech
+Java bindings.
+
+## Building the JNI library
+
+The JNI library can be built for desktop devices and Android
+devices. If you want to build the JNI library for Android devices,
+check out the Readme file of the `deepspeechandroid` project.
+
+There are two main configurations for the JNI library
+
+* CPU
+    * > Available on desktop and Android
+* Cuda
+    * > Only available on desktop devices
+
+To build a given configuration, modify the value of the `CUDA_CONFIGURATION`
+variable of the `CMakeLists.txt` build file.
+
+To build the cuda configuration go to the initial declaration of the variable
+in the build file and change it to either `TRUE` or `FALSE`
+depending on whether you want to build the CUDA or CPU configuration.
+
+```cmake
+set(CUDA_CONFIGURATION FALSE) # CPU configuration
+```
+
+```cmake
+set(CUDA_CONFIGURATION TRUE) # CUDA configuration
+```
+
+If you are building the library for android, _(Use Android Studio and also follow instructions of Readme of `deepspeechandroid` project)_
```

Comment:
We don't want to depend on a GUI tool like Android Studio.

Reply:
It's recommended (in my opinion), but not necessary.